cpp/cmake_modules/FindgRPCAlt.cmake

```diff
@@ -36,7 +36,6 @@ if(GRPCPP_PC_FOUND)
   # gRPC's pkg-config file neglects to specify pthreads.
   find_package(Threads REQUIRED)
   if(ARROW_GRPC_USE_SHARED)
-    set(GRPCPP_LINK_LIBRARIES ${GRPCPP_PC_LINK_LIBRARIES} Threads::Threads)
```

Comment:
Could you keep `GRPCPP_PC_LINK_LIBRARIES`?

`set(GRPCPP_LINK_LIBRARIES ${GRPCPP_PC_LINK_LIBRARIES})`

Reply:
Ah I missed the `_PC_`  I thought it was just old append syntax, sorry.